packages/frontend/src/views/create-event.vue

```diff
@@ -209,6 +232,20 @@ export default defineComponent({
       selectedTimes.value.splice(index, 1)
     }
 
+    const rawInput = ref('')
+
+    const updateMarkdown = debounce((e: Event): void => {
+      rawInput.value = (e.target as HTMLSpanElement).innerText
+    }, 500)
+
+    const displayedDescription = computed<string>(() => {
+      return markdown.value.render(rawInput.value)
+    })
+
+    const openMarkdownRef = () => {
+      window.open('https://markdown-it.github.io/')
```

Comment:
there is a vulnerability going on with using `window.open` without noopener attribute, you should do like this
https://stackoverflow.com/a/63627759/12386405

Reply:
ah forgot this, sorry